pkgs/by-name/be/beatsabermodmanager/package.nix

```diff
@@ -18,12 +18,12 @@
 }:
 
 buildDotnetModule rec {
-  pname = "BeatSaberModManager";
+  pname = "beatsabermodmanager";
```

Comment:
I don't see any point in changing the pname.

Reply:
pname must not contain uppercase letters, see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming.

Reply:
That seems rather arbitrary?